pkg/controller/networkpolicy/networkpolicy_controller.go

```diff
@@ -1146,6 +1183,19 @@ func podToGroupMember(pod *v1.Pod, includeIP bool) *controlplane.GroupMember {
 	return memberPod
 }
 
+func nodeToGroupMember(node *v1.Node) (member *controlplane.GroupMember) {
+	member = &controlplane.GroupMember{Node: &controlplane.NodeReference{Name: node.Name}}
```

Comment:
When getting some of the Node IPs failed, the group member includes the Node name only.